CppCoreGuidelines.md

```diff
@@ -6896,7 +6896,7 @@ not using this (over)general interface in favor of a particular interface found
 ##### Enforcement
 
 * Look for classes with lots of members that do nothing but throw.
-* Flag every use of a nonpublic base class `B` where the derived class `D` does not override a virtual function or access a protected member in `B`, and `B` is not one of the following: empty, a template parameter or parameter pack of `D`, a class template specialized with `D`.
+* Flag every use of a non-public base class `B` where the derived class `D` does not override a virtual function or access a protected member in `B`, and `B` is not one of the following: empty, a template parameter or parameter pack of `D`, a class template specialized with `D`.
```

Comment:
M-W has "nonpublic", but no "nonprivate" (earlier in this PR).
The standard avoided both. I'd say "nonpublic" is a common enough word.

Reply:
Thank you for the feedback!
As you mentioned "nonpublic" looks better option to me as well!

The result is different in dictionaries and so I will change it into "nonpublic" soon

#### nonvirtual

* In the stdandard (N4842)
  * 0 times
* In dictionaries
  * Google : O
  * Oxford : X
  * Collins : O
  * Dictionary.com : O
  * Merriam-Webster : O
  * Macmillan : X
  * Cambridge : X
  * Longman : X

Reply:
Hyphens are removed in nopublic by https://github.com/isocpp/CppCoreGuidelines/pull/1564/commits/2c112d7e233bbb49fea52d9fd1295003b4e0b941

Reply:
I resolve this conversation
If you have any suggestion please let me know !